### ISSUE TITLE
[macos] Error: ENOENT: no such file or directory, stat '/Users/kevin/Codes/geojson/.DS_Store'

### ISSUE THREAD

**Issue by kevinresol** [lix-pm/switchx#14]
Fails at the `statSync` call here: https://github.com/lix-pm/switchx/blob/62c7550b5033787765a2e659825ca8fe3b20f8fc/src/switchx/Switchx.hx#L103

There is no `.DS_Store` in the folder, it is strange that `readDirectory()` returns it.

**Comment by back2dos** [comment]
Yeah, that `.DS_Store` is a Mac thing that I see on Windows whenever I unzip an archive zipped on MacOS ^^

**Comment by kevinresol** [comment]
Probably should ignore filenames beginning with `.`

**Comment by back2dos** [comment]
Yeah, sure. What I don't get is why the `/Users/kevin/Codes/geojson/.DS_Store` (which is a lib, right?) is checked in `officialInstalled` (which is supposed to find official Haxe versions).

**Comment by kevinresol** [comment]
I am using https://github.com/lix-pm/travlix to run the tests, not sure if that is the reason

**Comment by back2dos** [comment]
Uhm, I'll gladly put a check at the line you've indicated, but I really do wonder how the path could ever be something that looks even remotely like `/Users/kevin/Codes/geojson/...` ^^

**Comment by kevinresol** [comment]
Fixed by https://github.com/lix-pm/haxeshim/commit/39fe9a3d26f1b18cb8a05c1ccf102b3a928b4a8b

**Comment by kevinresol** [comment]
Actually I think the bug should be fixed somewhat more upstream, that `.DS_Store` should not be recognized as a "version" when iterating the haxe version folder.

Probably adding `v.charCodeAt(0) != '.'.code &&` here: https://github.com/lix-pm/switchx/blob/0a59c7f59b7182fc55180c7314a3cfe71a1e31e7/src/switchx/Switchx.hx#L105
